tests/x11regressions/firefox/firefox_downloading.pm

```diff
@@ -36,6 +36,7 @@ sub dl_location_switch {
 }
 
 sub dl_save {
+    my ($self) = @_;
     my ($link) = @_;
```

Comment:
I'm pretty sure that won't do what you expect. This will initialize $link to be always empty